docs/ldml/tr35-numbers.md

```diff
@@ -95,7 +95,7 @@ The LDML specification is divided into the following parts:
 
 ## <a name="Numbering_Systems" href="#Numbering_Systems">Numbering Systems</a>
 
-```xml
+```dtd
```

Comment:
While editing this file, many validation issues were found in the editor that I was using. The changes outside of the RBNF section primarily involve these types of changes, including grammar and spelling errors.